QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.WebSocket.cs

```diff
@@ -70,6 +71,13 @@ public partial class TDAmeritradeBrokerage
             { typeof(OrderRouteMessage), new XmlSerializer(typeof(OrderRouteMessage))}
         };
 
+        /// <summary>
+        /// We keep userPrincipals in first request when Login to system
+        /// Then we need some param to subscribe\unsubscribe in channels
+        /// Also, we decrease amount requests
+        /// </summary>
+        private UserPrincipalsModel userPrincipals;
```

Comment:
Not respecting code style -> `_userPrincipals`

Reply:
I renamed to correct style type.